.aws/deploy/backend-task-definition.staging.json

```diff
@@ -0,0 +1,238 @@
+        },
+        { "name": "JWT_SECRET", "valueFrom": "STAGING_JWT_SECRET" },
+        {
+          "name": "MAX_NUM_OTP_ATTEMPTS",
+          "valueFrom": "STAGING_MAX_NUM_OTP_ATTEMPTS"
+        },
+        {
+          "name": "MOCK_AMPLIFY_DOMAIN_ASSOCIATION_CALLS",
+          "valueFrom": "STAGING_MOCK_AMPLIFY_DOMAIN_ASSOCIATION_CALLS"
+        },
+        { "name": "MUTEX_TABLE_NAME", "valueFrom": "STAGING_MUTEX_TABLE_NAME" },
+        {
+          "name": "NETLIFY_ACCESS_TOKEN",
+          "valueFrom": "STAGING_NETLIFY_ACCESS_TOKEN"
+        },
+        { "name": "NODE_ENV", "valueFrom": "STAGING_NODE_ENV" },
+        { "name": "OTP_EXPIRY", "valueFrom": "STAGING_OTP_EXPIRY" },
+        { "name": "OTP_SECRET", "valueFrom": "STAGING_OTP_SECRET" },
+        {
+          "name": "OUTGOING_QUEUE_URL",
+          "valueFrom": "STAGING_OUTGOING_QUEUE_URL"
+        },
+        { "name": "POSTMAN_API_KEY", "valueFrom": "STAGING_POSTMAN_API_KEY" },
+        {
+          "name": "POSTMAN_SMS_CRED_NAME",
+          "valueFrom": "STAGING_POSTMAN_SMS_CRED_NAME"
+        },
+        {
+          "name": "STAGING_SSH_PUBLIC_KEY",
+          "valueFrom": "STAGING_SSH_PUBLIC_KEY"
+        },
+        {
+          "name": "STAGING_SSH_PRIVATE_KEY",
+          "valueFrom": "STAGING_SSH_PRIVATE_KEY"
+        }
+      ],
+      "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+          "awslogs-group": "/aws/elasticbeanstalk/cms-backend-prod-node18/var/log/web.stdout.log",
+          "awslogs-region": "ap-southeast-1",
+          "awslogs-stream-prefix": "ecs"
+        }
+      }
+    },
+    {
+      "name": "dd-agent",
+      "image": "public.ecr.aws/datadog/agent:latest",
+      "portMappings": [
+        {
+          "containerPort": 8126,
+          "hostPort": 8126,
+          "protocol": "tcp"
+        }
+      ],
+      "essential": true,
+      "environment": [
+        {
+          "name": "TZ",
+          "value": "Asia/Singapore"
+        },
+        {
+          "name": "DD_APM_NON_LOCAL_TRAFFIC",
+          "value": "true"
+        },
+        {
+          "name": "ECS_FARGATE",
+          "value": "true"
+        },
+        {
+          "name": "DD_APM_ENABLED",
+          "value": "true"
+        },
+        {
+          "name": "DD_SITE",
+          "value": "datadoghq.com"
+        },
+        {
+          "name": "DD_ENV",
+          "value": "prod"
+        },
+        {
+          "name": "DD_SERVICE",
+          "value": "isomer"
+        },
+        {
+          "name": "DD_TAGS",
+          "value": "service:isomer"
+        },
+        {
+          "name": "DD_AGENT_MAJOR_VERSION",
+          "value": 7
+        },
+        {
+          "name": "DD_LOGS_INJECTION",
+          "value": true
+        },
+        {
+          "name": "DD_TRACE_STARTUP_LOGS",
+          "value": true
+        },
+        {
+          "name": "DD_API_KEY",
+          "valueFrom": "<DD_API_KEY>"
```

Comment:
Do we need the <>?

Reply:
this is used as a marker for `sed` to do a replcaement inside `aws_deploy.yml`

Reply:
got it